authority/admin/api/acme.go

```diff
@@ -49,7 +54,7 @@ func (h *Handler) requireEABEnabled(next nextHTTP) nextHTTP {
 
 // provisionerHasEABEnabled determines if the "requireEAB" setting for an ACME
 // provisioner is set to true and thus has EAB enabled.
-func (h *Handler) provisionerHasEABEnabled(ctx context.Context, provisionerName string) (bool, error) {
+func (h *Handler) provisionerHasEABEnabled(ctx context.Context, provisionerName string) (bool, *admin.Error) {
```

Comment:
Why return a specific error type from this method? 99.99% of the time we return golang default error. Just want to make sure that if we're going to do something different, that there's a very good reason.

Reply:
Good spot. It must've been left over from an earlier iteration. `error` is fine in this case.